pkg/redfish/redfish-foxconn.go

```diff
@@ -333,4 +333,28 @@ func discoverFoxconnENetInterfaces(s *EpSystem) {
 			}
 		}
 	}
+}
+
+// Determines if chassis name is a Foxconn chassis
+func isFoxconnChassis(chassis string) bool {
```

Comment:
Are these URLs only present on foxconn systems?
It seems like the check below should logically be something like: (isFoxconn() && isIgnorableFoxconnChassis(chassis))

Reply:
I check for OpenBMC by looking for presence of this url /redfish/v1/Chassis/BMC_0 in the /redfish/v1/Chassis output.

https://github.com/Cray-HPE/hms-hmcollector/blob/master/cmd/hmcollector/rf.go#L82-L94
https://github.com/Cray-HPE/hms-hmcollector/blob/master/cmd/hmcollector/rf.go#L82-L94

This is not an issue if those url's are not used or needed in other BMC implementations

Reply:
I did an inventory on all of the dev systems I have access to and it feels safe to consider them unique.  Here's the list (I'll also post in the Jira):

Unique River chassis names:  1, Self, RackMount

- HPE DL325: 1
- HPE DL325 v2: 1
- HPE DL385: 1
- HPE XL270d: 1
- Cray H262-Z63-YF (probably DL325 or DL385): 1
- Gigabyte R272-Z30-00: Self
- Cray R272-Z30-00 (probably Gigabyte): Self
- Cray R272-Z30-YF (probably Gigabyte): Self
- Cray H262-Z63-YF (probably Gigabyte): Self
- Intel S2600BPB: RackMount
- Intel S2600WFT: RackMount
- Cray S2600BPB (probably Intel): RackMount
- Cray S2600WFT (probably Intel): RackMount

Unique Mountain chassis names:  Node0, Node1, Mezz0, Mezz1, Mezz2, Mezz3, Enclosure

- WNC: Node0, Node1, Mezz0, Mezz1, Enclosure
- CNC: Node0, Node1, Mezz2, Mezz3, Enclosure
- BlancaPeakNC: Node0, Mezz1, Mezz3, Enclosure
- BardPeakNC: Node0, Mezz0, Mezz1, Enclosure
- ParryPeakNC: Node0, Mezz0, Mezz2, Enclosure
- GrizzlyPkNodeCard: Node0, Node1, Mezz0, Mezz1, Mezz2, Mezz3, Enclosure

Reply:
Additionally, I think it would be good to log when we skip a chassis so will add that and push up a commit.